main.py

```diff
@@ -1,26 +1,23 @@
 from os import getenv
 from telegram import Update
 from telegram.ext import ApplicationBuilder, MessageHandler, CallbackContext, filters
-import json
-import random
+from json import load
+from random import randrange
 
 
 def get_new_user_name(user: dict) -> str:
     #fallback does not work properly
     return f"@{user['username']}" if user['username'] != '' else user['first_name']
 
 def generate_welcome(new_member) -> str:
```

Comment:
Missing input type suggestion, it should be `User` or `dict`

Reply:
using dict for now